src/utils/Configuration.ts

```diff
@@ -117,6 +117,14 @@ class Configuration {
         return this.config.testNumberinitialValue;
     }
 
+    /**
+     * Retrieves the trailer id initial value
+     * @returns any
+     */
+    public getTrailerIdInitialValue(): any {
```

Comment:
This may not be needed. It is a remnant of the old test number implementation and no longer needed as we are now taking seed value.